lib/ansible/inventory/manager.py

```diff
@@ -206,15 +206,22 @@ def parse_sources(self, cache=False):
                 parse = self.parse_source(source, cache=cache)
                 all_parsed_results.append(parse)
 
-        # ensure all inventories parsed returned true
+        # ensure all parsed inventories returned true
         if all(all_parsed_results):
             # do post processing
             self._inventory.reconcile_inventory()
         else:
             if C.INVENTORY_UNPARSED_IS_FAILED:
-                raise AnsibleError("No inventory was parsed or errors detected, please check your configuration and options.")
+                raise AnsibleError("Errors detected during parsing, please check your configuration and options.")
```

Comment:
The word "inventory" should probably still be part of the error message just to make it more specific what sort of configuration/options to check.